packages/nimble-components/src/icon-base/tests/icons.stories.ts

```diff
@@ -1,6 +1,8 @@
 import type { Meta, StoryObj } from '@storybook/html';
 import { html, repeat } from '@microsoft/fast-element';
 import { DesignSystem } from '@microsoft/fast-foundation';
+// Workaround to force Chromatic TurboSnap to re-run this test when icons are changed
+import '../icon-metadata';
```

Comment:
Was looking at this again and had the notion that the icon-metadata is just for tests, effectively for this story.
I think we should move it into the tests folder to make sure product code doesn't add a dependency to it.

Would you be willing to move it in the tests dir as part of this PR?